site/docs/v1/tech/apis/identity-providers/google.adoc

```diff
@@ -11,182 +11,24 @@ description: APIs for creating, retrieving, updating and disabling the Google id
-
-This API is intended to be used if you want to build your own login page and you have added the Google login button to your own login page and you then need to complete the login with FusionAuth.
-
-For example, if you built your own login page, you could add a Login with Google button and complete the Google authentication. When you complete the Google authentication you will have been returned an id token from Google. Using this API you can pass that id token returned from Google to FusionAuth and we will complete the login workflow and reconcile the user to FusionAuth.
-
-The user does not need to exist yet in FusionAuth to utilize this API. The token returned from Google will be used to retrieve the user's email address, if that user does not yet exist in FusionAuth the user will be created. If create registration has been enabled for this identity provider and the user does not yet have a registration for this application, a registration will be automatically created for the user. The user will be assigned any default roles configured for the application. If create registration has not been enabled, a registration will not be created if one does not yet exist. This last scenario is useful if you wish to manually provision users and then subsequently allow them to login with Google.
-
-=== Request
-
-[.api-authentication]
-link:/docs/v1/tech/apis/authentication#no-authentication-required[icon:unlock[type=fas]] Complete Google Login
-[.endpoint]
-.URI
---
-[method]#POST# [uri]#/api/identity-provider/login#
---
-
-==== Request Headers
-
-[.api]
-include::docs/v1/tech/apis/identity-providers/_x-forwarded-for-header.adoc[]
-include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-scoped-operation-row-only.adoc[]
-
-:loginProvider: Google
-include::docs/v1/tech/apis/identity-providers/_identity-provider-login-request-body.adoc[]
-:loginProvider!:
-
-=== Response
-
-The response for this API contains the User object.
-
-:loginProvider: Google
-include::docs/v1/tech/apis/identity-providers/_identity-provider-login-response-body.adoc[]
-:loginProvider!:
+The Google identity provider type will use the Google OAuth v2.0 login API. It will also provide a `Login with Google` button on FusionAuth’s login page that will direct a user to the Google login page.
+
+This identity provider will call Google’s API to load the user's `email` and `preferred_username` and use those as `email` and `username` to lookup or create a user in FusionAuth depending on the linking strategy configured for this identity provider. Additional claims returned by Google can be used to reconcile the user to FusionAuth by using a Google Reconcile Lambda.
+
+FusionAuth will also store the Google `refresh_token` returned from the Google API in the link between the user and the identity provider. This token can be used by an application to make further requests to Google APIs on behalf of the user.
```

Comment:
Is this correct? I believe we are currently either storing the `id_token` when using a redirect, or the token returned by Google when using popup which is likely an `id_token` as well.

Reply:
You are correct.  They are both `id_tokens`.  I'll update the doc.